README.md

```diff
@@ -385,33 +385,43 @@ USAGE
   $ chectl server:update
 
 OPTIONS
-  -a, --installer=helm|operator|minishift-addon                                Installer type
-  -h, --help                                                                   show CLI help
+  -a, --installer=helm|operator|minishift-addon
+      Installer type
 
-  -n, --chenamespace=chenamespace                                              [default: che] Kubernetes namespace where
-                                                                               Eclipse Che server is supposed to be
-                                                                               deployed
+  -b, --domain=domain
```

Comment:
To me, using of `-b` short key is weird. Could you please explain logic behind it? Why not `-d` for **d**omain?

Reply:
I have no idea.

Reply:
I suppose from the very beginning short aliases where generated in alphabetic order.
+1 to start cleaning them up, start deprecation of weird ones and introduce new more logical ones.